tutor/commands/upgrade/common.py

```diff
@@ -43,6 +43,46 @@ def upgrade_from_nutmeg(context: click.Context, config: Config) -> None:
     )
 
 
+def get_mysql_change_authentication_plugin_query(config: Config) -> str:
+    """
+    Helper function to generate queries to upgrade the authentication plugin of MySQL users
+    By default, only the ROOT and OPENEDX users are upgraded
+    If any loaded plugins have database user configurations defined in the format:
+        <plugin>_MYSQL_USERNAME
+        <plugin>_MYSQL_PASSWORD
+    These users are also upgraded
+    """
+    loaded_plugins = list(plugins.iter_loaded())
+
+    def generate_mysql_authentication_plugin_update_query(
+        username: ConfigValue, password: ConfigValue, host: str
+    ) -> str:
+        return f"ALTER USER '{username}'@'{host}' IDENTIFIED with caching_sha2_password BY '{password}';"
+
+    host = "%"
```

Comment:
why is the host set to %?

Reply:
The host for [users created](https://github.com/overhangio/tutor/blob/a97a7b0abdb1ae73df2f0769a2609e6767b40075/tutor/templates/jobs/init/mysql.sh#L20) by Tutor is always set to '%'. The users created by MySQL itself uses the 'localhost' host.